tests/test_run_reference_vmm.py

```diff
@@ -61,8 +61,10 @@ def default_ubuntu_elf():
 
 def default_disk():
     return s3_download(
-        resource_type="disk",
-        resource_name="ubuntu-focal-rootfs.ext4",
+        resource_type="disk",        
+        resource_tags={
+            "platform_type":"x86_64"
```

Comment:
Here we should just read the platform on which we're running and use that one instead. We can also do this in a subsequent PR in which we also update the tests. The goal would be to reuse as many functions as possible on both platforms.

Reply:
I was planning to do it when we port of the tests from x86.

Reply:
Sounds good.